db/migrate/20201130142541_change_notifications_guid_not_null.rb

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Style/FrozenStringLiteralComment: Missing frozen string literal comment.

Reply:
Hush, there's not even a string in this file.